node_type: eagerx_core.node/ProcessNode
default:
  rate:
  inputs:
  outputs:
  single_process: True
  launch_locally: True
  launch_file: '$(find eagerx_core)/launch/rxnode.launch'
  test_arg: 'TEST ARGUMENT'

inputs:
  tick:
    msg_type: std_msgs.msg/UInt64
    repeat: all
  in_1:
    msg_type: std_msgs.msg/UInt64
    repeat: all

outputs:
  out_1:
    msg_type: std_msgs.msg/UInt64
